lib/realtime/monitoring/erl_sys_mon.ex

```diff
@@ -11,7 +11,8 @@ defmodule Realtime.ErlSysMon do
     :busy_dist_port,
     :busy_port,
     {:long_gc, 250},
-    {:long_schedule, 100}
+    {:long_schedule, 100},
+    {:long_message_queue, true}
```

Comment:
What is "long" here? This could log way too many times.

In Logfare we do: `{:long_message_queue, {0, 1_000}}`